app/views/business_cases/_table.html.haml

```diff
@@ -58,6 +55,15 @@
     %span.loading-spinner
       = image_tag 'wait.gif'
 
+  - elsif business_case_rows.empty?
+    .panel.panel-danger
+      .panel-heading Something went wrong with the business case calculation.
+      .panel-body
+        Please submit this error to the ETMoses&nbsp;
+        %a{ href: 'https://github.com/quintel/etmoses/issues' }> issues page
```

Comment:
> we might find a way to forward DelayedJob exceptions

Yes. Maybe [this](https://github.com/benjaminoakes/delayed-plugins-airbrake) is worth looking at?